apps/question-the-docs/backend/ai/utils/text.py

```diff
@@ -15,22 +17,60 @@ class TextChunker:
     def __call__(self, src, files, cache):
         return pd.concat([self._slide_window(src, f, cache) for f in files])
 
+    def _get_code_block_lines(self, lines, non_ended_code_block):
+        lines = lines.to_numpy().tolist()
```

Comment:
Nitpick: Can we make this a bit cleaner ie something that directly converts to a list.

Reply:
sure make sense